.github/workflows/pr_build.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
       - uses: actions/setup-node@v4
         with:
-          node-version: "22"
+          node-version: 22
```

Comment:
Should be a string to match all other repos, "22"

Reply:
fixed